client/.prettierrc.json

```diff
@@ -1,4 +1,15 @@
 {
-	"singleQuote": true,
```

Comment:
This is out of the scope of this PR. We are already working on a prettier configuration in another PR, and it is adjusted to the styles this project is using. Could you restore the code to the style that was already in place?

Reply:
Let me try. I do have a prettier extension that automatically. I'll make changes to this PR with the reverted codebase in some time.

Reply:
Great thanks!